shared/engine/index.js

```diff
@@ -41,8 +41,9 @@ class EngineChannel {
     return yield Saga.takeFromChannelMap(this._map, key)
   }
 
-  *race(options: ?{timeout?: number, racers?: Object}): Generator<any, any, any> {
+  *race(options: ?{timeout?: number, removeNs: boolean, racers?: Object}): Generator<any, any, any> {
     const timeout = options && options.timeout
+    const removeNs = (options && options.removeNs) || false
```

Comment:
I guess I found it confusing that it was mapping the keys to different results. I can change nbd